css-three-column-layout/styles.css

```diff
@@ -0,0 +1,60 @@
+body {
+  background-color: #0090e0;
+  font-family: Lato, Arial, Helvetica, sans-serif;
+}
+
+.container {
+  max-width: 1000px;
+  margin: auto;
+  background-color: #0090e0;
+  color: white;
+}
+
+.column-third {
+  height: 110px;
+  width: 110px;
+  margin: 10px 90px;
+  padding: -30px;
```

Comment:
Negative `padding` or `margin` is usually an indicator that you made an element too small. It is _highly_ unlikely to be the proper solution to a problem.